pkg/ruler/ruler.go

```diff
@@ -116,6 +115,8 @@ type Config struct {
 
 	EnableQueryStats bool `yaml:"query_stats_enabled" category:"advanced"`
 
+	Querier QuerierConfig `yaml:"query-frontend"`
```

Comment:
1. Should be:
```suggestion
	Querier QuerierConfig `yaml:"query_frontend"`
```

2. OTOH, why the yaml key is query frontend, and the struct field is querier? Can we use same name everywhere?